src/util.test.ts

```diff
@@ -24,22 +24,24 @@ describe("Util", (): void => {
 
     const mockedExec = async (
       error: Error | null,
-      stdout: string | null = null,
-      stderr: string | null = null
+      stdout: string = "Stdout",
+      stderr: string = ""
     ): Promise<void> => {
       child_process.exec.mockImplementationOnce(
         (_command: any, _options: any, callback: any): any => {
           callback(error, { stdout, stderr });
         }
       );
-      await util.execBashCommand(COMMAND);
+      const output = await util.execBashCommand(COMMAND);
 
       expect(core.info).nthCalledWith<[string]>(1, COMMAND);
       expect(child_process.exec).lastCalledWith(
         COMMAND,
         { shell: "/usr/bin/bash" },
         expect.anything()
       );
+      if (error) expect(output).toBe(stderr);
```

Comment:
This is only passing by coincidence; we expect `output` to be `""` when `error` is not `null`. It will probably be easier to move the asserts on `output` to the top level to avoid branching.